Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: docker
+
+TAG ?= mirantis/k8s-appcontroller
+
+docker: vendor Makefile
+	docker build -t $(TAG) .
+
+vendor: Makefile
+	glide install --strip-vendor
```

Comment:
So we run 'glide install'  twice while building a docker img: here and in the dockerfile. THat does not look right. We probably should do something with it.